src/resolver.ts

```diff
@@ -171,12 +171,32 @@ export interface ResolveArgs {
 
 const SKIP_POSITIONAL_DEFAULT = -1
 
+/**
+ * Tracks which arguments were explicitly provided by the user.
+ *
+ * Each property indicates whether the corresponding argument was explicitly
+ * provided (true) or is using a default value or not provided (false).
+ *
+ * @template A - The arguments schema type
```

Comment:
sorry for jsdoc no-guideline. 🙇 
args-tokens repo respect jsdoc, not tsdoc.
let's remove this line.

Reply:
fixed in: 7e60e98